haddock-api/src/Haddock/Interface.hs

```diff
@@ -165,6 +167,7 @@ createIfaces verbosity flags instIfaceMap mods = do
 processModule :: Verbosity -> ModSummary -> [Flag] -> IfaceMap -> InstIfaceMap -> Ghc (Maybe Interface)
 processModule verbosity modsum flags modMap instIfaceMap = do
   out verbosity verbose $ "Checking module " ++ moduleString (ms_mod modsum) ++ "..."
+  liftIO $ hSetEncoding stderr utf8
```

Comment:
Okay, I got it. I just should have installed standalone cabal install from https://www.haskell.org/cabal/download.html ...
Maybe fixed the problem! Thanks!